build_sysext

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Flatcar Linux. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Script to generate sysext. See systemd-sysext(8). Prerequisite that you've run build_packages and build_image
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+
+DEFAULT_GROUP=developer
+DEFAULT_DEVCONTAINER_BINHOST="${SETTING_BINPKG_SERVER_PROD}"
+
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/cros_vm_constants.sh" || exit 1
+
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "Board for which the sysext was built"
+
+# include upload options
+. "${BUILD_LIBRARY_DIR}/release_util.sh" || exit 1
+
+FLAGS_HELP="USAGE: build_sysext [flags] [package to build].
+This script is used to build a Flatcar sysext. 
+
+Examples:
+
+build_sysext --board=amd64 python - builds python sysext for amd64 architecture.
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV}"
+
+# N.B.  Ordering matters for some of the libraries below, because
+# some of the files contain initialization used by later files.
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+
+BUILD_DIR="${DEFAULT_BUILD_ROOT}/images/${BOARD}/${IMAGE_SUBDIR}"
+
+# export SYSEXTNAME=$2
+
+# # with my 1 line patch before the rms in build_library/prod_image_util.sh (mksquashfs "${root_fs_dir}" "${BUILD_DIR}/configroot" "${BUILD_DIR}"/flatcar_production_image_sysext.sqsh)
+sudo mkdir ${BUILD_DIR}/fs-root
```

Comment:
Here and in the lines below we should quote variables.
We should also support rerunning and handle the case that this folder exists or that the mount is still set up.
The best would be to run `umount` first to be safe and also set up a `trap` to `umount` when the script exits.